### ISSUE TITLE
Breaking the provider config from 7.4 to 7.8

### ISSUE THREAD

**Issue by slzmruepp** [jfrog/terraform-provider-artifactory#729]
**Describe the bug**
After release of 7.8.0 we get the issue:
```

  			&{
  				... // 5 identical fields
  				Computed:        false,
  				Sensitive:       false,
- 				DescriptionKind: s"PLAIN",
+ 				DescriptionKind: s"MARKDOWN",
  				Deprecated:      false,
  			},
  			&{
  				Name:            "url",
  				Type:            s"tftypes.String",
- 				Description:     "",
+ 				Description:     "Artifactory URL.",
  				Required:        false,
  				Optional:        true,
  				Computed:        false,
  				Sensitive:       false,
- 				DescriptionKind: s"PLAIN",
+ 				DescriptionKind: s"MARKDOWN",
  				Deprecated:      false,
  			},
  		},
  		BlockTypes:  nil,
  		Description: "",
  		... // 2 identical fields
  	},
  }


╷
│ Error: an object with 4 attributes is required (3 given)
│ 
│   with provider["registry.terraform.io/jfrog/artifactory"],
│   on main.tf line 65, in provider "artifactory":
│   65: provider "artifactory" {
│ 
╵
##[error]Terraform command 'validate' failed with exit code '1'.
##[error]╷
│ Error: an object with 4 attributes is required (3 given)
│ 
│   with provider["registry.terraform.io/jfrog/artifactory"],
│   on main.tf line 65, in provider "artifactory":
│   65: provider "artifactory" {
```

The provider config looks as following:
```
    artifactory = {
      source = "jfrog/artifactory"
      version = ">= 7.4.0"
    }
...
provider "artifactory" {
  url = "https://${var.REGISTRYURL}/artifactory/"
}
```
The Token is submitted as ENV Variable named:
 JFROG_ACCESS_TOKEN: $(tf-jfrog-pat)
```
 We had to pin the provider back to 7.4.0 to get the pipeline running again.

**Requirements for and issue**
- [ ] A description of the bug
- [ ] A fully functioning terraform snippet that can be copy&pasted (no outside files or ENV vars unless that's part of the issue). **If this is not supplied, this issue will likely be closed without any effort expended.**
- [ ] Your version of artifactory (you can `curl` it at `$host/artifactory/api/system/version`
- [ ] Your version of terraform
- [ ] Your version of terraform provider

**Expected behavior**
Non breaking provider configuration

**Additional context**
Terraform v1.4.6

**Comment by danielmkn** [comment]
@slzmruepp, do you have `api_key` set in your vars?

**Comment by marcgarlandjr** [comment]
I'm having a similar issue. Downgrading to 7.7.0 fixes it. I have a similar block to OP with the url parameter set. ARTIFACTORY_API_KEY is set through the environment.

**Comment by danielmkn** [comment]
It's fixed, releasing later today.

**Comment by danielmkn** [comment]
@marcgarlandjr, please don't use the API key, it's deprecated.

**Comment by danielmkn** [comment]
Fix released in 7.10.0. Thanks for the ticket!
